lib/vagrant-service-manager/installer.rb

```diff
@@ -5,15 +5,15 @@ def initialize(type, machine, env, options)
         @type = type
         @machine = machine
         @env = env
-        @box_version = options[:box_version]
+        @options = { type: @type }.merge(options)
```

Comment:
Is this not a bit odd? You pass _type_ and _option_ to this method, but then add _@type_ to the options as well? Would it not make more sense to add the type of the options before and just pass the options parameter?